mlflow/environment_variables.py

```diff
@@ -652,3 +652,12 @@ def get(self):
 _MLFLOW_IN_CAPTURE_MODULE_PROCESS = _BooleanEnvironmentVariable(
     "MLFLOW_IN_CAPTURE_MODULE_PROCESS", False
 )
+
+#: Use DatabricksSDKModelsArtifactRepository when registering and loading models to and from
+#: Databricks UC. This is required for SEG(Secure Egress Gateway) enabled workspaces and helps
+#: eliminate models exfiltration risk associated with temporary scoped token generation used in
+#: existing model artifact repo classes. This flag is expected to be removed in 2025 Q1 and mlflow
```

Comment:
```suggestion
#: existing model artifact repo classes. mlflow
```

I don't think we should reveal the internal plan.